Need to drop ECDH from algorithm list when building on Solaris,
since our GnuTLS build disables it in
userland/components/gnutls-3/patches/02_remove_elliptical.patch

--- a/common/rfb/SSecurityTLS.cxx	2017-03-11 06:54:49.040654074 +0000
+++ b/common/rfb/SSecurityTLS.cxx	2017-03-11 06:55:09.835235271 +0000
@@ -158,7 +158,7 @@
 
 void SSecurityTLS::setParams(gnutls_session_t session)
 {
-  static const char kx_anon_priority[] = ":+ANON-ECDH:+ANON-DH";
+  static const char kx_anon_priority[] = ":+ANON-DH";
 
   int ret;
   char *prio;
--- a/common/rfb/CSecurityTLS.cxx	2017-03-11 06:54:14.514328881 +0000
+++ b/common/rfb/CSecurityTLS.cxx	2017-03-11 06:54:38.542837546 +0000
@@ -192,7 +192,7 @@
 
 void CSecurityTLS::setParam()
 {
-  static const char kx_anon_priority[] = ":+ANON-ECDH:+ANON-DH";
+  static const char kx_anon_priority[] = ":+ANON-DH";
 
   int ret;
   char *prio;
